### PR TITLE
Add TopK columns

### PR DESCRIPTION
**Pull request by ajtritt**
- Add TopKProbabilities, a column for storing the top k class probabilities
- Add TopKClasses, a column for storing the top k classes

### CODE REVIEW

File: ml/ml_table.yaml [hunk 1]
```diff
@@ -48,6 +48,16 @@ datasets:
...
 
+- data_type_def: TopKProbabilities
+  data_type_inc: SupervisedOutput
+  dtype: float
+  doc: A column for storing the top k class probabilities. Probabilities should be ordered in descending order.
+
+- data_type_def: TopKClasses
+  data_type_inc: SupervisedOutput
+  dtype: float
+  doc: A column for storing the top k classes
```
Comment (rly) [inline ml/ml_table.yaml:59]: Can you give an example of data for this column? Why is it a float?

Comment (rly) [reply ml/ml_table.yaml:59]: Should TopKProbabilities and TopKClasses always be present together? And should len(TopKProbabilities) == len(TopKClasses)

Comment (ajtritt) [reply ml/ml_table.yaml:59]: TopKClasses should be an int. I will fix that. 

They don't always need to be stored together.

Comment (ajtritt) [reply ml/ml_table.yaml:52]: For context, top-k classes gets used for top-k accuracy calculation. Checkout the [scikit-learn function](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.top_k_accuracy_score.html) for a more detailed explanation

File: ml/ml_table.yaml [hunk 2]
```diff
@@ -48,6 +41,16 @@ datasets:
...
 
+- data_type_def: TopKProbabilities
+  data_type_inc: SupervisedOutput
+  dtype: float
+  doc: A column for storing the top k class probabilities. Probabilities should be ordered in descending order.
+
+- data_type_def: TopKClasses
+  data_type_inc: SupervisedOutput
+  dtype: int
+  doc: A column for storing the top k classes
```
Comment (rly) [inline ml/ml_table.yaml:52]:
```suggestion
  doc: A column for storing the top k classes, used for top-k accuracy calculation.
```

**Review by rly — APPROVED** [review]
